src/sql/parts/notebook/notebook.contribution.ts

```diff
@@ -18,7 +18,12 @@ import { IInstantiationService } from 'vs/platform/instantiation/common/instanti
 import { NotebookInput, NotebookInputModel, notebooksEnabledCondition } from 'sql/parts/notebook/notebookInput';
 import { NotebookEditor } from 'sql/parts/notebook/notebookEditor';
 import { CommandsRegistry } from 'vs/platform/commands/common/commands';
+import { INotebookProviderRegistry } from 'sql/services/notebook/notebookRegistry';
 
+const DEFAULT_NOTEBOOK_FILETYPE = 'IPYNB';
+const Extensions = {
```

Comment:
Please import Extensions from the notebookRegistry file instead. This should then be available and avoids duplication. That's what we do in all other cases where there's a registry ID like this